### DISCUSSION TITLE
Is `wastedassign` still broken?

### DISCUSSION THREAD

**Discussion by stdedos** [golangci/golangci-lint#3396] [Q&A] [answered]
_Apologies, I decided to avoid spamming https://github.com/golangci/golangci-lint/issues/2649_

Even though the repo "seems inactive" since June 25, someone has probably demonstrated this working?
https://github.com/sanposhiho/wastedassign/issues/41#issuecomment-1211675563

Does [golangci-lint](https://github.com/golangci/golangci-lint) have integration tests that https://github.com/sanposhiho/wastedassign still fails?

**Comment by ldez** [comment]
I have re-run my test suite to check that

**Comment by Antonboom** [comment, marked as answer]
Covered by https://github.com/golangci/golangci-lint/pull/3689
